src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -368,6 +373,8 @@ bool SimulatedTestFixture::tickTest(Duration simulation_time_step, Duration ai_t
         *friendly_world = friendly_sensor_fusion.getWorld().value();
         *enemy_world    = enemy_sensor_fusion.getWorld().value();
 
+        LOG(VISUALIZE) << *createWorld(*friendly_world);
```

Comment:
pretty sure this doesn't make it to the thunderscope

Reply:
yeah good callout, I added a visualize_cpp_test mode as a stopgap until we get rid of this fixture in the near future

Reply:
I'm getting 
```
===== 2022.04.20 11:04:23 =====
Traceback (most recent call last):
  File "/home/jonathan/.cache/bazel/_bazel_jonathan/126329019c7d97b3885b38e9ee5a3b31/execroot/__main__/bazel-out/k8-fastbuild/bin/software/thunderscope/thunderscope.runfiles/__main__/software/thunderscope/thunderscope.py", line 605, in <module>
    proto_unix_io.attach_unix_receiver(*arg)
  File "/home/jonathan/.cache/bazel/_bazel_jonathan/126329019c7d97b3885b38e9ee5a3b31/execroot/__main__/bazel-out/k8-fastbuild/bin/software/thunderscope/thunderscope.runfiles/__main__/software/thunderscope/proto_unix_io.py", line 134, in attach_unix_receiver
    listener = ThreadedUnixListener(
  File "/home/jonathan/.cache/bazel/_bazel_jonathan/126329019c7d97b3885b38e9ee5a3b31/execroot/__main__/bazel-out/k8-fastbuild/bin/software/thunderscope/thunderscope.runfiles/__main__/software/networking/threaded_unix_listener.py", line 39, in __init__
    self.server = socketserver.UnixDatagramServer(
  File "/usr/lib/python3.8/socketserver.py", line 452, in __init__
    self.server_bind()
  File "/usr/lib/python3.8/socketserver.py", line 466, in server_bind
    self.socket.bind(self.server_address)
FileNotFoundError: [Errno 2] No such file or directory
```
when running with visualize_cpp_test

Reply:
hmm what command are you running? 

I can't repro w/ `./tbots.py run thunderscope --visualize_cpp_test`

Reply:
discussed offline: deleting /tmp/tbots fixed the issue

the cleanup code already exists to remove old sockets, ill just add it in the troubleshooting docs if that doesn't fix it